### DISCUSSION TITLE
[auth][error] CredentialsSignin, when return null for CredentialsProvider in the authorize function

### DISCUSSION THREAD

**Discussion by yashrajtalokar** [nextauthjs/next-auth#11045] [Help]
How can I customize my invalid login messages 
![image](https://github.com/nextauthjs/next-auth/assets/70269192/f003b17f-4dab-4988-a6b0-0be6d511ba13)

But I still face this log at my server

**Comment by Qodestackr** [comment]
This lib is a mess.
